include/SSVOpenHexagon/Components/CPlayer.hpp

```diff
@@ -43,7 +43,7 @@ class CPlayer
     void drawDeathEffect(HexagonGame& mHexagonGame);
 
 public:
-    CPlayer(const sf::Vector2f& mStartPos, const float swapCooldown) noexcept;
+    CPlayer(const sf::Vector2f& mPos, const float swapCooldown) noexcept;
```

Comment:
```suggestion
    CPlayer(const sf::Vector2f& mStartPos, const float swapCooldown) noexcept;
```

No need for this change

Reply:
Oh my sorry! I don't remember why I did that but will totally revert

Reply:
Actually I now do remember why I did it.
I made the name consistent with the .cpp naming
```
CPlayer::CPlayer(const sf::Vector2f& mPos, const float swapCooldown) : [...]
```
I made both mStartPos hoping that is fine